examples/sycl/dbscan_batch.py

```diff
@@ -118,8 +118,7 @@ def cpu_context():
                                result_gpu.coreObservations)
 
     with cpu_context():
-        # TODO: investigate hangs with data as sycl_buffer
-        sycl_data = data
+        sycl_data = sycl_buffer(data)
```

Comment:
Is it fast now? Was dramatically slow before.

Reply:
Yes, because of changes in `sycl_buffer` and because of usm memory backend in oneDAL